.github/workflows/check_diff.yml

```diff
@@ -0,0 +1,17 @@
+name: Allow only Additions
+
+on:
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  check-diff:
```

Comment:
this is the name displayed on each PR, maybe let's make it more descriptive `only-additions` or something like that.

Reply:
Ignore me